addons/centered/README.md

```diff
@@ -7,7 +7,7 @@ Storybook Centered Decorator can be used to center components inside the preview
 ### Usage
 
 ```sh
-npm install @storybook/addon-centered --save-dev
+npm install @storybook/addon-centered --save
```

Comment:
That's not true. Mostly people are using storybook in their dev deps, since Storybook is a tool for their projects, but not the dependency they rely within the application. That means all the SB packages are also, mostly, should be installed into dev deps.

Reply:
I see, yeah it depends on how someone uses storybook right?

If they feel it's a deliverable / deployable unit, then it's part of their regular dependencies.

TBH for most apps I've worked with it doesn't really matter too much, for libs obviously it matters a lot.

I suggest to keep it with `--save-dev` as @igor-dv suggests.

What do you think @grp06 ?